bob-common/src/bob_client.rs

```diff
@@ -172,6 +172,27 @@ pub mod b_client {
             }
         }
 
+        pub async fn delete(&self, key: BobKey, options: DeleteOptions) -> DeleteResult {
+            let node_name = self.node().name().to_owned();
+            let mut client = self.client.clone();
+            self.metrics.delete_count();
+            let timer = BobClientMetrics::start_timer();
+            let message = DeleteRequest {
+                key: Some(BlobKey { key: key.into() }),
+                options: Some(options),
+            };
+            let mut req = Request::new(message);
+            self.set_credentials(&mut req);
+            self.set_timeout(&mut req);
+            self.metrics.delete_timer_stop(timer);
+            if client.delete(req).await.is_ok() {
+                Ok(NodeOutput::new(node_name, ()))
+            } else {
+                self.metrics.delete_error_count();
+                Err(NodeOutput::new(node_name, Error::timeout()))
```

Comment:
Why `Error::timeout()` is used here? In the functions above the error returned from call is used (`Err(NodeOutput::new(node_name, e.into()))`)